app/controllers/results_controller.rb

```diff
@@ -241,8 +243,11 @@ def show
         end
 
         # Totals
-        data[:assignment_max_mark] =
-          result.is_a_review? ? assignment.pr_assignment.max_mark(:peer) : marks_map.map { |h| h['max_mark'] }.sum
+        if result.is_a_review?
+          data[:assignment_max_mark] = assignment.pr_assignment.max_mark(:peer_visible)
+        else
+          data[:assignment_max_mark] = marks_map.map { |h| h['max_mark'] }.sum
```

Comment:
Modify this to call `assignment.max_mark` (I don't know why this wasn't used before).